docs/development.md

```diff
@@ -0,0 +1,46 @@
+# Development
```

Comment:
I was able to run the system on bare metal, but when I tried to run the docker on my workstation, I got this error message:

```
  File "/usr/lib/python3/dist-packages/compose/service.py", line 1579, in get_container_data_volumes
    container.image_config['ContainerConfig'].get('Volumes') or {}
    ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
KeyError: 'ContainerConfig'
make: *** [makefile:11: up] Error 1
```

Any ideas?

I don't normally run docker on my workstation and might be missing something obvious.